src/Analysis/GWASV2/Components/Covariates/ContinuousCovariates.jsx

```diff
@@ -13,35 +21,56 @@ const ContinuousCovariates = ({ dispatch, handleClose, submitButtonLabel }) => {
 
   return (
     <React.Fragment>
-      <Covariates selected={selected} handleSelect={setSelected} />
-      <button
-        type='button'
-        style={{ marginLeft: 5 }}
-        onClick={() => {
-          dispatch(formatSelected(selected));
-          handleClose();
-        }}
-      >
-        {submitButtonLabel}
-      </button>
-      <button
-        type='button'
-        onClick={() => {
-          handleClose();
-        }}
-      >
-        Cancel
-      </button>
+      <div className='GWASUI-flexRow' style={{ width: '950px' }} data-tour='name'>
+        <div>
+          <Covariates selected={selected} handleSelect={setSelected} />
+          <button
+            type='button'
+            style={{ marginLeft: 5 }}
+            onClick={() => {
+              dispatch(formatSelected(selected));
+              handleClose();
+            }}
+          >
+            {submitButtonLabel}
+          </button>
+          <button
+            type='button'
+            onClick={() => {
+              handleClose();
+            }}
+          >
+          Cancel
+          </button>
+        </div>
+        <div style={{ paddingLeft: '30px' }}>
+          {selected
+            ? (
+              <PhenotypeHistogram
+                selectedStudyPopulationCohort={selectedStudyPopulationCohort}
+                selectedCovariates={selectedCovariates}
+                outcome={outcome}
+                selectedContinuousItem={selected}
+              />
+            )
+            : 'Select a variable'}
```

Comment:
Do you think it would be more clear if this conditionally rendered

```Select an outcome phenotype``` or ```Select a covariate phenotype``` based on the currentStep?

Reply:
Good point! I've adjusted the text to match the rest of the UI. It now states:

"'Select a concept to render its corresponding histogram'"